### DISCUSSION TITLE
How do I prevent external users from accessing Authentik Dashboard?

### DISCUSSION THREAD

**Discussion by CliveTran** [goauthentik/authentik#9367] [Q&A]
Hi everyone,
So let's say I have a simple social media app with couple thousand users. All I want is just providing users the login/logout mechanism to obtain JWT using OIDC Provider and forward the token to my APIs. I don't want external users to be able to access the Overview screen below

![image](https://github.com/goauthentik/authentik/assets/55139389/c6bfa0be-6bab-42aa-82ab-93ac6db9abae)

how can I achieve this?

**Comment by BeryJu** [comment]
As somewhat of a workaround you can use https://docs.goauthentik.io/docs/flow/examples/snippets#redirect-current-flow-to-another-url to redirect the user after their authentication/enrollment flow. This will be more easier after https://github.com/goauthentik/authentik/pull/9257 is merged
